concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetElectionInfo.java

```diff
@@ -0,0 +1,51 @@
+package com.concordium.sdk.examples;
+
+import com.concordium.sdk.ClientV2;
+import com.concordium.sdk.Connection;
+import com.concordium.sdk.Credentials;
+import com.concordium.sdk.exceptions.BlockNotFoundException;
+import com.concordium.sdk.exceptions.ClientInitializationException;
+import com.concordium.sdk.requests.BlockHashInput;
+import com.concordium.sdk.responses.blockinfo.BlockInfo;
+import com.concordium.sdk.responses.election.ElectionInfo;
+import lombok.var;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+@Command(name = "GetElectionInfo", mixinStandardHelpOptions = true)
+public class GetElectionInfo implements Callable<Integer> {
+    @Option(
+            names = {"--endpoint"},
+            description = "GRPC interface of the node.",
+            defaultValue = "http://localhost:20002")
```

Comment:
I would recommend using 20000 as the port. 20002 is not a default anywhere, so setting it to that is not really that useful.

Reply:
done